lib/Elastica/Transport/AbstractTransport.php

```diff
@@ -99,14 +99,13 @@ public static function create($transport, Connection $connection, array $params
             } else {
                 $transport = ucfirst($transport);
             }
-
-            $className = 'Elastica\\Transport\\'.$transport;
-
-            if (!class_exists($className)) {
```

Comment:
It does, If $transport is not a AbstractTransport ([Line 112](https://github.com/ruflin/Elastica/pull/1169/files#diff-dee909e4e66d40dd2dea7d7147d4bbc7L112)), It will throw InvalidException ([Line 119](https://github.com/ruflin/Elastica/pull/1169/files#diff-dee909e4e66d40dd2dea7d7147d4bbc7L119)).